### PR TITLE
Add dockerignore file

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

Add `.dockerignore` file.

## Why

Lighter images to make the CI faster and improve the network load.

### CODE REVIEW

File: .dockerignore
Comment (wbeuil) [inline .dockerignore:4]: You could easily add `.github`, `LICENCE` and `README.md` to this list since we don't need them too.

**Review by wbeuil — CHANGES_REQUESTED** [review]
Hello @KeisukeYamashita, thanks for your contribution.

I just found out that for public repo `Secrets are not passed to workflows that are triggered by a pull request from a fork`, that's why the Action failed.

**Comment by KeisukeYamashita** [comment]
@wbeuil Yeah, forked PRs can't access to secrets and it needs maintainers approval.

Thanks for your reply.

**Comment by wbeuil** [comment]
We added the .dockerignore in the latest version of the github-action. I will close this issue. Thanks for your contribution.

**Pull request closed without merging**
